packages/react-components/react-combobox/src/components/Dropdown/useDropdown.tsx

```diff
@@ -148,8 +177,37 @@ export const useDropdown_unstable = (props: DropdownProps, ref: React.Ref<HTMLBu
       elementType: 'span',
     }),
     placeholderVisible: !baseState.value && !!props.placeholder,
+    showClearButton,
     ...baseState,
   };
 
+  const onClearButtonClick = useEventCallback(
+    mergeCallbacks(state.clearButton?.onClick, (ev: React.MouseEvent<HTMLButtonElement>) => {
+      ev.stopPropagation();
```

Comment:
again, not sure why `stopPropagation` is necessary

Reply:
Indeed, removed.